src/vcpkg/commands.generate-message-map.cpp

```diff
@@ -215,31 +212,55 @@ namespace vcpkg::Commands
         }
         std::sort(messages.begin(), messages.end(), MessageSorter{});
 
-        bool has_incorrect_comment = false;
+        bool has_errors = false;
         LocalizedString format_string_parsing_error;
         Json::Object obj;
         for (Message& msg : messages)
         {
+            if (msg.name != "ErrorMessage" && Strings::case_insensitive_ascii_starts_with(msg.value, "error:"))
```

Comment:
These are the new lints. Note that I removed the switch to turn the lints off; we control all these inputs.